lib/Utils/Backends.pm

```diff
@@ -47,40 +59,84 @@ our %EXPORT_TAGS = (
 
 # Use it after SUT boot finish, as it requires ssh connection to SUT to
 # interact with SUT, including window and serial console
+
+=head2 use_ssh_serial_console
+
+Selects the root-ssh and saves it to SERIALDEV
+
+=cut
+
 sub use_ssh_serial_console {
     select_console('root-ssh');
     $serialdev = 'sshserial';
     set_var('SERIALDEV', $serialdev);
     bmwqemu::save_vars();
 }
 
+=head2 is_remote_backend
+
+remote backend
```

Comment:
Please try to explain better what the method is doing...

Reply:
Also for the others. Something like:

"Returns true if the current instance is running as a X backend"

Reply:
changed